spec/models/spree/user_spec.rb

```diff
@@ -201,4 +201,36 @@
       expect(user.flipper_id).to eq "Spree::User;42"
     end
   end
+
+  describe "#disable!" do
```

Comment:
It would also be good to add a system spec because the translation from checkbox values to the database could fail.

Reply:
Currently facing timeout issues with Cuprite/Ferran gems, checking out the issue.